data/kbs-storage/default/sigstore-config/test

```diff
@@ -0,0 +1,4 @@
+docker:
+    quay.io/kata-containers/confidential-containers:
+        sigstore: file:///etc/containers/quay_verification/signatures
+        sigstore-staging: file:///etc/containers/quay_verification/signatures
```

Comment:
Are these paths populated via attestation-agent as well or do they need to be pre-provisioned to initrd?

Reply:
These paths will not be created at the runtime.

If `simple-signing` is actually used (in `policy.json`), these paths and the related signature files need to be pre-provisioned to initrd. (As we only support to look for signature files from local filesystem now for `simple-signing`)

If `simple-signing` is not used. These paths will not be accessed during the runtime, so we do not need to create them or pre-provision to  initrd.